extension.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "DFRawFunctions",
+	"version": "1.7",
+	"author": "Quietust",
+	"url": "http://dwarffortresswiki.org/index.php/User:Quietust",
+	"description": "Dwarf Fortress Raw parser functions",
+	"type": "parserhook",
+	"requires": {
+		"MediaWiki": ">= 1.33"
+	},
+	"Hooks": {
+		"ParserFirstCallInit": "DFRawFunctions::efDFRawFunctions_Initialize"
+	},
+	"AutoloadClasses": {
+		"DFRawFunctions": "DFRawFunctions.body.php"
+	},
+	"ExtensionMessagesFiles": {
+		"DFRawFunctionsMagic": "DFRawFunctions.i18n.magic.php"
+	},
+	"config": {
+		"DFRawEnableDisk": {
+			"value": false
+		},
+		"DFRawPath": {
+			"value": ""
+		},
+		"DFRawVersion": {
+			"value": ""
```

Comment:
Could you add a `description` field to explain what this is? My guess is the default version if no version is specified.

Reply:
Of course https://github.com/DF-Wiki/DFRawFunctions/pull/10/commits/d779adfdf68c1645ba9051607c1900dbc3490462